storage/src/vespa/storage/persistence/asynchandler.h

```diff
@@ -13,8 +13,11 @@ namespace spi {
 }
 struct PersistenceUtil;
 
+/**
+ * Handle async operations that uses a sequenced executor
```

Comment:
Nitpick: Missing '.' at the end. On the next sentence as well.

Reply:
Fixed